indigo_api/models/tasks.py

```diff
@@ -49,11 +49,11 @@ def get_queryset(self):
 
 
 class Task(models.Model):
-    OPEN = 'open'
-    PENDING_REVIEW = 'pending_review'
-    CANCELLED = 'cancelled'
-    DONE = 'done'
-    BLOCKED = 'blocked'
+    OPEN = __('open')
```

Comment:
these are used in the db and must not be translated, otherwise the codes in the db will differ